clash-cores/test/Test/Cores/I2C/Config.hs

```diff
@@ -51,88 +52,84 @@ configT s0 (rst,ena,cmdAck,rxAck,al) = do
   sNext <- if rst then pure confInit else case confStateM of
     CONFena
       | ena && not done
-      -> pure s { i2cConfStateM = CONFaddr }
+      -> pure s { i2cConfStateM = CONFaddr
+                , i2cConfClaim  = True
+                }
       | done
       -> do display "done"
             pure s
 
     CONFaddr
-      -> pure s { i2cConfStateM = CONFaddrAck
-                , i2cClaim = True
-                , i2cWrite = True
-                , i2cDin   = unpack i2cSlvAddr
-                }
+      -> do
+        display $ "CONFaddr, writing: " <> showHex i2cSlvAddr ""
+        pure s { i2cConfStateM = CONFaddrAck
+               , i2cConfOp     = Just (WriteData (unpack i2cSlvAddr))
+               }
 
     CONFaddrAck
       | success
-      -> do display "CONFaddrAck"
-            pure s { i2cConfStateM = CONFreg
-                   , i2cWrite = False
+      -> if rxAck then do
```

Comment:
Are these changes made because the ack made combinational in the another commit? Perhaps mention that in the commit message then.